stellar-asset-spec/src/tests/events.rs

```diff
@@ -10,6 +10,7 @@ use soroban_sdk::{
 struct Contract;
 
 #[test]
+#[allow(deprecated)]
```

Comment:
Before merging once we're settled on the impl can we update all the tests so that they call both `all` and the new fn while both are being supported?